### ISSUE TITLE
iOS 13.4 下 好像没有效果

### ISSUE THREAD

**Issue by webjyh** [alexiscn/WXNavigationBar#2]
新升级了 Xcode  Version 11.4  在模拟器 iPhone 11   iOS 13.4 版本上  WXNavigationBar  好像无效了，升级之后才有的，没升级之前是没问题的

**Comment by alexiscn** [comment]
It seems a bug of iOS 13.4 when debugging in Xcode 11.4. But it will take effect on next launch.

I will add manually setup in next version

**Comment by alexiscn** [comment]
Please update to version 2.0.1. And add following code in AppDelegate

```swift
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
    //....
    WXNavigationBar.setup()
}
``` 

I also noticed that `preferredStatusBarStyle` in ViewController will not work when debugging in Xcode 11.4 but worked when not debugging. So I think it's a bug of Xcode 11.4.  And hope it will be fixed in Xcode 11.4.1.

**Comment by webjyh** [comment]
好的，谢谢~
